exercises/list-ops/test/test_list_ops.c

```diff
@@ -9,6 +11,33 @@ void tearDown(void)
 {
 }
 
+static char *print_list(list_t * list)
+{
+   char *array = (char *)malloc(MAX_STRING_LEN * sizeof(char));
```

Comment:
It's best practice to not cast the value returned from `malloc()`

Reply:
👍 

Done